tcms/testruns/forms.py

```diff
@@ -19,6 +20,10 @@ class Meta:
 
     manager = UserField()
     default_tester = UserField(required=False)
+    start_date = DateTimeField()
+    stop_date = DateTimeField()
+    planned_start = DateTimeField()
+    planned_stop = DateTimeField()
```

Comment:
These changes look good, ideally they should come after the test is implemented to demonstrate that the new changes actually fix a problem. At the moment that's unclear and the current failure is due to other things.

Leave the changes as they are and update the commit log to reflect that all these 4 fields now accept both input formats for timestamps.